src/azure-cli/azure/cli/command_modules/servicebus/Operation/Namespace_custom_file.py

```diff
@@ -0,0 +1,234 @@
+            }
+        })
+
+    if encryption_config:
+        command_args_dict.update({"encryption": {
+            "key_vault_properties": encryption_config,
+            "key_source": "Microsoft.KeyVault",
+            "require_infrastructure_encryption": require_infrastructure_encryption
+        }})
+    return Create(cli_ctx=cmd.cli_ctx)(command_args=command_args_dict)
+
+
+def cli_add_encryption(cmd, resource_group_name, namespace_name, encryption_config,
+                       require_infrastructure_encryption=None):
+    from azure.cli.command_modules.servicebus.aaz.latest.servicebus.namespace import Update
+    from azure.cli.command_modules.servicebus.aaz.latest.servicebus.namespace import Show
+
+    servicebusnm = Show(cli_ctx=cmd.cli_ctx)(command_args={
+        "resource_group": resource_group_name,
+        "namespace_name": namespace_name
+    })
+    key_vault_object = []
+    for col in encryption_config:
+        key_vault_object.append(col)
+
+    if 'encryption' in servicebusnm:
+        for col in servicebusnm['encryption']['keyVaultProperties']:
+            vault_object = create_keyvault_object(col)
+            if vault_object not in key_vault_object:
+                key_vault_object.append(vault_object)
+        if require_infrastructure_encryption is None:
+            require_infrastructure_encryption = servicebusnm['encryption']['requireInfrastructureEncryption']
+
+    return Update(cli_ctx=cmd.cli_ctx)(command_args={
+        "resource_group": resource_group_name,
+        "namespace_name": namespace_name,
+        "encryption": {
+            "key_source": "Microsoft.KeyVault",
+            "key_vault_properties": key_vault_object,
+            "require_infrastructure_encryption": require_infrastructure_encryption
+        }
+    })
+
+
+def cli_remove_encryption(cmd, resource_group_name, namespace_name, encryption_config):
+    from azure.cli.command_modules.servicebus.aaz.latest.servicebus.namespace import Update
+    from azure.cli.command_modules.servicebus.aaz.latest.servicebus.namespace import Show
+
+    servicebusnm = Show(cli_ctx=cmd.cli_ctx)(command_args={
+        "resource_group": resource_group_name,
+        "namespace_name": namespace_name
+    })
+
+    key_vault_object = []
+
+    for col in servicebusnm['encryption']['keyVaultProperties']:
+        vault_object = create_keyvault_object(col)
+        key_vault_object.append(vault_object)
+    for col in encryption_config:
+        if col in key_vault_object:
+            key_vault_object.remove(col)
+    return Update(cli_ctx=cmd.cli_ctx)(command_args={
+        "resource_group": resource_group_name,
+        "namespace_name": namespace_name,
+        "encryption": {
+            "require_infrastructure_encryption": servicebusnm['encryption']['requireInfrastructureEncryption'],
+            "key_vault_properties": key_vault_object,
+            "key_source": "Microsoft.KeyVault"
+        }
+    })
+
```

Comment:
You can try subcommands generation in aaz-dev-tools

Reply:
Let's merge this PR, because all cmdlets are working fine.
Will try subcommands in upcoming PR for sure